src/utils/mac.c

```diff
@@ -23,7 +23,7 @@
 
 #include "utils/mac.h"
 
-inline int pg_scan_ether_addr(struct ether_addr *eth_addr, const char *buf)
+inline bool pg_scan_ether_addr(struct ether_addr *eth_addr, const char *buf)
 {
 	int ret;
 
```

Comment:
If you change the return type to bool, you need to change `return 0;` to `return false;`, just cleaner